haste.py

```diff
@@ -12,12 +10,12 @@ async def init(bot, modules):
         import aiohttp
     except ImportError:
         aiohttp = None
-        logging.warning('aiohttp module not available; #haste command disabled')
+        logging.warning("aiohttp module not available; #haste command disabled")
```

Comment:
Mostly everywhere else, single quotes are used, so I don't understand this change.